src/renderer/services/midgard/pools.ts

```diff
@@ -255,6 +256,39 @@ const createPoolsService = (
       RxOp.shareReplay(1)
     )
 
+  /**
+   * Loading queue to get all needed data for `AllPoolsState`
+   */
+  const loadAllPoolsStateData$ = (): AllPoolsStateLD => {
+    const poolAssets$: PoolAssetsLD = FP.pipe(
+      apiGetPoolsAll$,
+      // Filter out all unknown / invalid assets created from asset strings
+      liveData.map(A.filterMap(({ asset }) => FP.pipe(asset, assetFromString, O.fromNullable))),
+      // Filter pools by using enabled chains only (defined via ENV)
+      liveData.map(A.filter(({ chain }) => isEnabledChain(chain))),
+      RxOp.shareReplay(1)
+    )
+    const poolDetails$ = getPoolDetails$(poolAssets$)
+
+    return FP.pipe(
+      liveData.sequenceS({
```

Comment:
`sequenceS` is not needed here (there is just one stream to handle, `sequenceXYZ` is used for more than one...)